.github/workflows/release.yml

```
@@ -1,21 +1,25 @@
 name: Release
 
 on:
-  push:
-    tags: # Excludes generic vN tags
-      - v2.*
-      - v3.*
-      - v4.*
-      - v5.*
+  workflow_dispatch:
+    inputs:
+      Version:
+        description: 'Example: v1.2.3'
+        required: true
 
 jobs:
-  Changelog:
+  Release:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 50
-      - uses: notlmn/release-with-changelog@v1
+      - run: git tag ${{ github.event.inputs.Version }}
+      - uses: notlmn/release-with-changelog@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           exclude: '^Meta'
+      - name: Update major tag
+        run: |
+          MAJOR=$(echo ${{ github.event.inputs.Version }} | sed 's/\..*//')
+          git push origin HEAD:refs/tags/$MAJOR -f
```

```suggestion
          git push origin HEAD:refs/tags/$MAJOR --force
```

No short flags, `-f` vs `--force`!